INSTALL

```diff
@@ -0,0 +1,59 @@
+PREREQUISITES:
+--------------
+
+* bash
+* autotools (automake, autoconf, libtool)
+* make
+* c compiler
+* libjson-c: rt-app uses libjson-c to parse the .json file with the workload
+  description. You can either install the libjson-c2 and libjson-c-dev deb
+  packages or compile from source code available here:
+
+      https://github.com/json-c/json-c
+
+  The following sequence can be used to cross compile rt-app with static linkage
```

Comment:
This is for compiling json-c library, no?

Anyway, I'm actually thinking that, while we are at it, we could add steps for both compiling using dyn libraries and static ones.

Reply:
14s/rt-app/json-c/: you're right, i missed that one.

yes, could build both, static and dynamic json-c, will change accordingly.

do you actually want an INSTALL file? i just realised that maybe the installation instructions fit into the README.md (since it already has some...)?

Reply:
Very good point indeed!
I'd say we should actually live with a single README.in (the one that it's already there) and improve that.
Please move build info there and thanks for pointing this out!